### ISSUE TITLE
get_keyword result capped at 50 movie titles

### ISSUE THREAD

**Issue by WafaJohal** [alberanid/imdbpy#297]
#### Issue description

get_keyword seems to return a max of 50 movies. Why?

#### Steps to reproduce the issue

```
#!python
 dog_movies = imdb.get_keyword('dog')
cat_movies =  imdb.get_keyword('cat')

print(len(dog_movies))
print(len(cat_movies))
```

#### What's the expected result?

I would like it to return all the movies with the keyword.
Is there a way to change this cap?

**Comment by alberanid** [comment]
Hi @WafaJohal ,
it's because the parsed page is paginated (e.g.: https://www.imdb.com/search/keyword/?keywords=dog )

The `get_keyword` method supports two optional arguments:
* **results**=50, which apparently is now ignored by IMDb and always returns 50 titles per page
* **page**=1, to get more results

So it's easy to cycle until you get no results. However a keyword like **dog** has over 14k results, so I expect it to be extremely slow.

Hope this helps.

**Comment by BrandonKMLee** [comment]
Is this problem related to https://github.com/alberanid/imdbpy/issues/324 ?

**Comment by WafaJohal** [comment]
Yes I ended up using the start argument to loop over the pages and get the movies 50 at a time.
